Directory.Build.targets

```diff
@@ -59,29 +59,31 @@
     <RemoveDir Directories="$(UnityPackageProject)" />
   </Target>
 
-  <Target Name="CleanCocoaSDK" AfterTargets="Clean" Condition="Exists('$(SentryCocoaArtifactsDestination)')">
+  <Target Name="CleanCocoaSDK" AfterTargets="Clean" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
```

Comment:
Same 'run only when building this project' strategy to make sure this run only once.
Removed the old condition because it didn't add anything.